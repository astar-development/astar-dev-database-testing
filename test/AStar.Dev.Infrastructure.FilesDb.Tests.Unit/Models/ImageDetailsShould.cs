using AStar.Dev.Utilities;
using JetBrains.Annotations;
using Shouldly;

namespace AStar.Dev.Infrastructure.FilesDb.Models;

[TestSubject(typeof(ImageDetail))]
public class ImageDetailsShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new ImageDetail(1234, 5678).ToJson().ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedToString()
        => new ImageDetail(1234, 5678).ToString().ShouldMatchApproved();
}
