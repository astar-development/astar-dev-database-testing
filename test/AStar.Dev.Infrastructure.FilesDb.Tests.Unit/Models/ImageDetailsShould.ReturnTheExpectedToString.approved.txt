ImageDetail { Width = 1234, Height = 5678 }